### PR TITLE
Add master build specific badge to README.md

### PR DESCRIPTION
**Pull request by otawfik-ms**
@Microsoft/fsharp-compiler

### CODE REVIEW

**Comment by cston** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@
...
-|**vs2015**  ||| [![Build status](https://img.shields.io/appveyor/ci/KevinRansom/visualfsharp-radou/vs2015.svg)](https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/branch/vs2015) |
-|**coreclr** ||| [![Build status](https://img.shields.io/appveyor/ci/KevinRansom/visualfsharp-radou/coreclr.svg)](https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/branch/coreclr) |
+|**master**  |[![Build Status](http://dotnet-ci.cloudapp.net/buildStatus/icon?job=Microsoft_visualfsharp/master_debug_windows_nt_prtest)](http://dotnet-ci.cloudapp.net/job/Microsoft_visualfsharp/job/master_debug_windows_nt_prtest/)|[![Build Status](http://dotnet-ci.cloudapp.net/buildStatus/icon?job=Microsoft_visualfsharp/master_release_windows_nt_prtest)](http://dotnet-ci.cloudapp.net/job/Microsoft_visualfsharp/job/master_release_windows_nt_prtest/)|[![Build status](https://img.shields.io/appveyor/ci/KevinRansom/visualfsharp-radou/master.svg)](https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/branch/master) |
```
Comment (enricosada) [inline README.md]: _prtest is right? Prtest is not the job for building pr only ?

Comment (otawfik-ms) [inline README.md]: @enricosada Pending confirmation from @mmitche, it seems that jenkins encodes branch information in the same folder. We are looking into #973 first before I update/merge this one.

Comment (otawfik-ms) [inline README.md]: Submitted the following PR to add the other two branches:
https://github.com/dotnet/dotnet-ci/pull/179
Will wait for both of them to be pulled before updating this one.

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by otawfik-ms** [comment]
@mmitche any idea why this launched the coreclr branch job?

**Comment by mmitche** [comment]
@otawfik-ms Because coreclr has a general PR trigger addGithubPRTrigger vs. addGithubPRTriggerForBranch.  The netci should be integrated over to that branch

Also you should reference the jobs here:

**Pull request merged**